src/webapp/src/client/resources/en_US/friendRecommendation.json

```diff
@@ -0,0 +1,9 @@
+{
+  "submitTicket": "Recommend a friend to Simoona team",
```

Comment:
The purpose of this feature was not about recommendation to join Simoona team, but to join a company as a whole (in this case - Visma). Would be good to generalize it in some other way, like - "Recommend a friend to join our company".